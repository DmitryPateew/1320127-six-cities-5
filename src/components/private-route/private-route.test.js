import {expect, it} from "@jest/globals";
import Enzyme, {shallow} from 'enzyme';
import {shallowToJson} from 'enzyme-to-json';
import Adapter from 'enzyme-adapter-react-16';
import React from "react";
import {PrivateRoute} from "./private-route";

Enzyme.configure({adapter: new Adapter()});

it(`Should priveteRoute render correctly`, () => {
  const router = shallow(<PrivateRoute path={`/favorites`} exact={false} render={() => {
  }} authorizationStatus={`NO_AUTH`}/>);
  expect(shallowToJson(router)).toMatchSnapshot();
});
