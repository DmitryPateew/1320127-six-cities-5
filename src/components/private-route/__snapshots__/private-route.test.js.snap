// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should priveteRoute render correctly 1`] = `
<Route
  exact={false}
  path="/favorites"
  render={[Function]}
/>
`;
